### PR TITLE
Typing `lodash`.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context

This PR is adding type definitions for `lodash` and fixes the resulting
type errors.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx [hunk 1]
```diff
@@ -37,7 +38,7 @@ const { DecoratorsActions } = CombinedProvider.get('Decorators');
 const DecoratorsConfig = () => {
   const [streams, setStreams] = useState<Array<Stream> | undefined>();
   const [currentStream, setCurrentStream] = useState(DEFAULT_STREAM_ID);
-  const [decorators, setDecorators] = useState();
+  const [decorators, setDecorators] = useState<Array<Decorator>>();
```
Comment (linuspahl) [inline graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx:41]: In similar cases we type this as `Array<Decorator> | undefined`, but we can adjust this case when we enable `strictNullChecks`

**Review by linuspahl — APPROVED** [review]
Looks good to me!

File: graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx (hunk 1 above)
Comment (dennisoelkers) [reply graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx:41]: Good catch. I prefer to change it right now!

Comment (dennisoelkers) [reply graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx]: :white_check_mark:

Comment (dennisoelkers) [reply graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx]: :white_check_mark:

**Pull request merged**